.github/workflows/oidc-conformity.yml

```diff
@@ -13,8 +13,8 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 2
-          repository: ory/hydra
-          ref: master
+          repository: narg95/hydra
```

Comment:
This is a small hack :) to get the OpenID conformity test running in sync with the migration branch used in hydra PR.
This needs to be reverted before merging